wolfcrypt/src/rsa.c

```diff
@@ -1526,16 +1541,20 @@ static int RsaUnPad_OAEP(byte *pkcsBlock, unsigned int pkcsBlockLen,
         return BAD_FUNC_ARG;
     }
 
+#ifdef WOLFSSL_SMALL_STACK
```

Comment:
Are you checking `pkcsBlockLen` <= `RSA_MAX_SIZE/8 + RSA_PSS_PAD_SZ`?

Reply:
@dgarske, Adding another check would be redundant since we've `(pkcsBlockLen < (2 * (word32)hLen + 2))`.